docs/querying/functions.md

```diff
@@ -532,7 +532,7 @@ resets due to target restarts) are automatically adjusted for. Also, the
 calculation extrapolates to the ends of the time range, allowing for missed
 scrapes or imperfect alignment of scrape cycles with the range's time period.
 
-The following example expression returns the per-second rate of HTTP requests as measured
+The following example expression returns the per-second rate of HTTP requests averaged
```

Comment:
Maybe let's stay as consistent as possible and use the same wording as above, so no misunderstanding is possible that we are talking about the same thing as in the previous paragraph?

```suggestion
The following example expression returns the per-second average rate of HTTP requests
```